pytorch/docker-compose.yaml

```diff
@@ -187,3 +187,8 @@ services:
       - 8082:8082
       - 7070:7070
       - 7071:7071
+  hf-genai:
+    build:
+      target: hf-genai
+    extends: multinode
+    image: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-${PACKAGE_OPTION:-pip}-py${PYTHON_VERSION:-3.10}-ipex-${IPEX_VERSION:-2.3.0}-oneccl-inc-${INC_VERSION:-2.5.1}-hf-genai
```

Comment:
Is there a relevant transformers version that we should track?

Reply:
Even if we don't add that version to the tag, it makes sense to peg the version for `transformers`

Reply:
I think it would be good to track the version of transformers. Especially if you are trying to run a transformers example, you need to check out that exact tag to match the installed transformers.

Reply:
Updated the image name to track transformers version